staging/src/k8s.io/apiserver/pkg/endpoints/filters/impersonation.go

```diff
@@ -58,128 +58,139 @@ func WithImpersonation(handler http.Handler, a authorizer.Authorizer, s runtime.
 			return
 		}
 
-		// if groups are not specified, then we need to look them up differently depending on the type of user
```

Comment:
Undo all of these changes to this file.  Update `constrained_impersonation.go` so that it handles the legacy path itself.

Reply:
reverted the change. All is handled in constrained_impersonation.go